src/js/nav.js

```diff
@@ -40,6 +41,10 @@ $(function() {
 	// On scroll, determine which section is in view, and highlight it
 	$(document).on('scroll', function() {
 		if (scrolltimer) return;
+
+		// Fix navigation only in large layouts
+		var affixInCurrentLayout = /^(L|XL)$/.test(getCurrentLayout());
```

Comment:
I've found that it's better to specify all modules used in bower.json in case you remove one at some point and  then you lose the dependency. Also, I don't know if they'll appear in the dependencies bit of the registry if don't specify them.